scripts/generate_package_diffs_for_flavors/generate_package_diffs_for_flavors.py

```diff
@@ -258,11 +258,21 @@ def generate_dependency_diff_report_for_all_flavors(working_copy_1_root: Path,
     for flavor_path in Path(working_copy_1_root, "flavors").iterdir():
         if flavor_path.is_dir():
             relative_flavor_path = flavor_path.relative_to(working_copy_1_root)
-            diffs = compare_flavor(relative_flavor_path, working_copy_1_root, working_copy_1_name,
-                                   relative_flavor_path, working_copy_2_root, working_copy_2_name)
+            relative_flavor_path_2 = relative_flavor_path
+            if Path(working_copy_2_root).joinpath(relative_flavor_path).exists():
+                diffs = compare_flavor(relative_flavor_path, working_copy_1_root, working_copy_1_name,
+                                       relative_flavor_path, working_copy_2_root, working_copy_2_name)
+            else:
+                print(f"Please enter the path to the flavor to which we should compare '{relative_flavor_path}':")
+                relative_flavor_path_2 = Path(input())
+                if Path(working_copy_2_root).joinpath(relative_flavor_path_2).exists():
+                    diffs = compare_flavor(relative_flavor_path, working_copy_1_root, working_copy_1_name,
+                                           relative_flavor_path_2, working_copy_2_root, working_copy_2_name)
+                else:
```

Comment:
I believe the idea is to use that branch for new flavors, right?
Maybe add a comment...